raiden/transfer/views.py

```diff
@@ -292,20 +292,32 @@ def get_channelstate_for(
     if token_network and creator_address in token_network.channelidentifiers_to_channels or \
         token_network and partner_address in token_network.channelidentifiers_to_channels:
         channels = []
-        for channel_id in token_network.partneraddresses_to_channelidentifiers[partner_address]:
+        if partner_address:
```

Comment:
Why we need to do this differentiation?

Reply:
The normal behavior of this function is to search by partner_address (parameter in the function), i adapt this function to return a channel_state if you dont send the partner_address, so it'll return the first channel for the node. That "if" is necessary so we can make the  search correctly.

Reply:
Please re-review this, i created a function to get the first channel_state in case that channel_state is none